pal03377/Athena-New
# Add feedback editor to send feedbacks

### Motivation and Context



#47 added some changes that allows to send feedbacks for a whole submission at a time, we did not update the playground to consider this.
#61 introduced a feedback editor for a submission which is perfectly suited for the send feedbacks under module requests, also the feedback editor debug view accidentally got merged into main. 

### Description


- Remove feedback editor test debug view
- Remove feedback selector and send feedbacks as batch per submission
- Enable feedback editor for the send feedbacks section

### Steps for Testing


1. Open the playground and navigate to Send Feedbacks
2. Play around with it to check if anything is not working as intended

### Screenshots

<img width="1341" alt="image" src="https://github.com/ls1intum/Athena/assets/5898705/9324e7e3-756f-4ccf-9bc1-a523de1b787b">
